tests/unit/lms/views/api/onedrive_test.py

```diff
@@ -0,0 +1,8 @@
+from lms.views.api.onedrive import authorize
+
+
+class TestOneDriveAuthorizeView:
+    def test_it(self, pyramid_request):
+        template_variables = authorize(pyramid_request)
+
+        assert template_variables == {}
```

Comment:
You could make this a function (rather than a class method) and cram it onto one line:

```suggestion
def test_redirect_uri(pyramid_request):
    assert authorize(pyramid_request) == {}
```